bin/build-locales

```diff
@@ -39,5 +40,23 @@ poFiles.forEach((pofile) => {
     format: 'jed1.x',
     fuzzy: config.get('po2jsonFuzzyOutput'),
   });
-  fs.writeFileSync(jsonfile, json);
+
+  // Add the moment locale JS into our locale file, if one is available.
+  var defineLocale = null;
+  try {
+    const momentLocale = locale.replace('_', '-').toLowerCase();
+    defineLocale = new Function(
+      fs.readFileSync(
+        `./node_modules/moment/locale/${momentLocale}.js`, 'utf8'
+      )
+    );
+  } catch (e) {
+    if (locale !== 'en_US') {
+      console.info(`No moment i18n available for ${locale}; consider adding one or creating a mapping.`);
```

Comment:
I think you can use `oneLine` from common-tags. Pretty sure that got added back in recently.